_docs-sources/pipelines/maintain/adding-aws-accounts.md

```diff
@@ -0,0 +1,16 @@
+# Adding AWS accounts
+
+If your organization has a need for more than one `infrastructure-live` repository, there are two primary approaches to adding Pipelines support to additional repositories:
+
+1. Create a new `infrastructure-pipelines` repository (though named something different) for the new `infrastructure-live` repository
+1. Use a single `infrastructure-pipelines` repository for multiple `infrastructure-live` repositories.
```

Comment:
This is not currently supported. We have a story this sprint to determine how to best accomplish this.

Reply:
Ah, that's going to be pretty timely, actually. I suppose in the meantime, customers can always use one Pipelines per infra-live repo, though.